modules/oe_editorial_corporate_workflow/modules/oe_editorial_corporate_workflow_translation/tests/modules/oe_editorial_corporate_workflow_translation_test/oe_editorial_corporate_workflow_translation_test.info.yml

```diff
@@ -1,8 +1,8 @@
-name: 'OpenEuropa Editorial Corporate Workflow Translation Test'
-description: 'Test module for the OpenEuropa Editorial Corporate Workflow Translation'
+name: OpenEuropa Editorial Corporate Workflow Translation Test
+description: Test module for the OpenEuropa Editorial Corporate Workflow Translation
```

Comment:
The same as for the others. Please add the single quotes back where it's been removed and add them where missing.